Improved export dialog, added feature to bundle capture and analyzed …

chaneylc opened the pull request:
issue #41

chaneylc committed:
Improved export dialog, added feature to bundle capture and analyzed photos.

trife commented:
- If only one box is checked, export file doesn't need to be zipped and can just be the raw csv
- Rename sub-files to match the main zip container (i.e. `Output_2021_....._seeds.csv`)
- Put images into subfolders: `analyzed`, `captured`

chaneylc committed:
Merge remote-tracking branch 'origin/dev' into issue41_v2.1

# Conflicts:
#	onekk/src/main/res/values/strings.xml

chaneylc committed:
If only one box is checked, export file doesn't need to be zipped and can just be the raw csv
Rename sub-files to match the main zip container (i.e. Output_2021_....._seeds.csv) [currently has a hard-coded "temp" prefix]
Put images into subfolders: analyzed, captured
Fixed select all analysis not toggling properly.

trife commented:
- This currently breaks with the included sample since there's no capture/analyzed image. Might need to just run the sample through the entire pipeline on startup so that it generates a capture/analyzed image.

chaneylc committed:
Fixed sample images not saved correctly.

trife committed:
Merge branch 'dev' into issue41_v2.1

trife committed:
Updated changelog
